sycl/tools/get_device_count_by_type.cpp

```diff
@@ -74,13 +62,13 @@ static bool queryOpenCL(cl_device_type deviceType, cl_uint &deviceCount,
   iRet = clGetPlatformIDs(0, nullptr, &platformCount);
   if (iRet != CL_SUCCESS) {
     if (iRet == CL_PLATFORM_NOT_FOUND_KHR) {
-      msg = "ERROR: OpenCL error runtime not found";
-    } else {
-      std::stringstream stream;
-      stream << "ERROR: OpenCL error calling clGetPlatformIDs " << iRet
-             << std::endl;
-      msg = stream.str();
+      msg = "OpenCL error runtime not found";
+      return true;
     }
+    std::stringstream stream;
+    stream << "ERROR: OpenCL error calling clGetPlatformIDs " << iRet
+           << std::endl;
+    msg = stream.str();
     return false;
```

Comment:
Suggest treating all errors in the function like "OK OpenCL is broken, let's ignore it".